src/global/hooks.js

```diff
@@ -49,7 +46,40 @@ const checkCoursePermission = permission => async (context) => {
 	return context;
 };
 
+const joinChannel = (prefix, sufixId = '_id') => (context) => {
+	const { app, result, params } = context;
+
+	if (params.provider !== 'socketio') { return context; }
+	const { connections } = app
+		.channel(app.channels)
+		.filter(connection => connection.userId === params.user.id);
+	if (result[sufixId]) {
+		connections.forEach(connection => app.channel(`${prefix}/${result[sufixId]}`).join(connection));
+	} else if (result.data && Array.isArray(result.data)) {
+		result.data.forEach(element => connections
+			.forEach(connection => app.channel(`${prefix}/${element[sufixId]}`).join(connection)));
+	} else if (Array.isArray(result)) {
+		result.forEach(element => connections
+			.forEach(connection => app.channel(`${prefix}/${element[sufixId]}`).join(connection)));
+	}
```

Comment:
For readable:
* const join = connection => app.channel(`${prefix}/${result[sufixId]}`).join(connection)
connections.forEach(join)
* Between first and second block one blank line. 

If no condition true do nothing is what you want?